agent-bootstrap/src/main/java/io/opentelemetry/auto/instrumentation/api/SpanWithScope.java

```diff
@@ -19,7 +19,62 @@
 import io.opentelemetry.context.Scope;
 import io.opentelemetry.trace.Span;
 
-// intentionally (for now) not implementing Closeable since not clear what it would close
+/**
+ * This is deprecated.
+ *
+ * <p>Originally, we used {@code SpanWithScope} to pass the {@link Span} and {@link Scope} between
+ * {@code @Advice.OnMethodEnter} and {@code @Advice.OnMethodExit}, e.g.
+ *
+ * <pre>
+ *   &#64;Advice.OnMethodEnter(...)
+ *   public static CallDepth onEnter(
```

Comment:
Don't think this is the old pattern, though it's not a big deal to document in detail either I think.

Reply:
thx! bad copy-paste 😂